archive/p/pascal/README.md

```diff
@@ -4,17 +4,22 @@ Welcome to Sample Programs in Pascal!
 
 ## Sample Programs
 
+
 - [Capitalize in Pascal][capitalize]
 - [Even Odd in Pascal][even-odd]
 - [Fizz Buzz in Pascal][fizz-buzz]
 - [Hello World in Pascal][hello-world]
 - [Prime Number in Pascal][prime-number]
 - [Reverse String in Pascal][reverse-string]
+- [Factorial in Pascal](https://github.com/TheRenegadeCoder/sample-programs/issues/2219)
```

Comment:
The issue linked here should be an article issue that you create here: https://github.com/TheRenegadeCoder/sample-programs-website

Once you do that, any chance you could follow the current style? For example, the list is alphabetized and the links are stored below by string.

Reply:
I need help on this.

Process I have been following is:

Create an Issue to update README
Add Sample Program Name and place the Issue in brackets in README ==> Create PR 
Create an Issue to create NEW FILE  #2219 
Add Code ==> Create PR  #2220 


What I now see is README is modified


I am asked to create a new markdown file in sample-programs-website Repo, instead of placing 2219 Issue in brackets.

Is this the correct requirement?

Reply:
I'd take a quick look at the contributing doc: https://github.com/TheRenegadeCoder/sample-programs/blob/master/.github/CONTRIBUTING.md